apps/web/pages/[user]/[type].tsx

```diff
@@ -70,7 +93,7 @@ async function getUserPageProps(context: GetStaticPropsContext) {
   const { type: slug, user: username } = paramsSchema.parse(context.params);
   const ssg = await ssgInit(context);
 
-  const user = await prisma.user.findUnique({
+  const user = await prisma.user.findFirst({
     where: {
       /** TODO: We should standarize this */
       username: username.toLowerCase().replace(/( |%20)/g, "+"),
```

Comment:
**[Blocker - Org]**
The org context is missing here. I tried adding it but the problem is that this page is SSG and we don't have access to context.req. I think we would have to make it a regular SSR route to achieve this.
This is what happens https://www.loom.com/share/e9aa0b84ea9141e1a24d3940f567f1d1

Reply:
Made changes to use getServerSideProps for this file now.